docs/alert.md

```diff
@@ -3,35 +3,35 @@
 
 ```jsx
 import { Alert } from 'radiance-ui';
-const SampleContentComponent = () => (<div>Your info is updated!  <a href='http://google.com'> Click here to see changes</a></div>);
+const SampleContentComponent = () => <div>Click <a href='https://google.com'>here</a> to go to Google!</div>;
 
 <Alert.Container>
   <Alert
-    text="Your photos were uploaded successfully!"
+    content="Your photos were uploaded successfully!"
     type="success"
     duration={4}
     onExit={() => {}}
   />
   <Alert
-    text="Your photos were uploaded successfully!"
+    content="Your photos were uploaded successfully!"
     type="success"
     duration="sticky"
     onExit={() => {}}
   />
   <Alert
-    text="Support has been notified."
+    content="Support has been notified."
     type="info"
     duration="sticky"
     onExit={() => {}}
   />
   <Alert
-    text="Oops! We couldn't update your information!"
+    content="Oops! We couldn't update your information!"
     type="danger"
     duration="sticky"
     onExit={() => {}}
   />
   <Alert
-	content=<SampleContentComponent/>
+	content= <SampleContentComponent/>
```

Comment:
still getting some weird spacing here. Maybe your ESlint is not configured properly? Also, doesn't it need curly braces `content={}`